lib_tflite_micro/src/inference_engine.cc

```diff
@@ -128,10 +128,12 @@ int inference_engine_load_model(inference_engine *ie, uint32_t model_bytes,
   uint8_t *kTensorArena = (uint8_t *)ie->memory_primary;
   int kTensorArenaSize = ie->memory_primary_bytes;
 
+  bool isModelInPrimaryMemory = false;
```

Comment:
I agree, the nomenclature is confusing. I will add a JIRA issue so that we can tidy up the primary/secondary terminology. I think it is also confusing to customers of our tools.